http/config.go

```diff
@@ -85,6 +85,12 @@ func (cfg *ServerConfig) SetAllowCredentials(flag bool) {
 	cfg.corsOpts.AllowCredentials = flag
 }
 
+func (cfg *ServerConfig) SetAllowHeaders(headers ...string) {
+	cfg.corsOptsRWMutex.Lock()
+	defer cfg.corsOptsRWMutex.Unlock()
+	cfg.corsOpts.AllowedHeaders = append(cfg.corsOpts.AllowedHeaders, headers...)
```

Comment:
This should either be called `AddAllowedHeaders` or it should set (not append) the allowed headers.